package/MDAnalysis/coordinates/XYZ.py

```diff
@@ -387,7 +388,7 @@ def open_trajectory(self):
 
         return self.xyzfile
 
-    def Writer(self, filename, n_atoms=None, **kwargs):
+    def Writer(self, filename, atoms=None, **kwargs):
```

Comment:
this would bring XYZ out of line with all the other Writers, so it's probably best to leave this as `n_atoms`